Makefile

```
@@ -72,3 +76,7 @@ updatereadme:
 .PHONY: tidy
 tidy:
 	@$(foreach dir,$(MODULE_DIRS),(cd $(dir) && go mod tidy) &&) true
+
+.PHONY: vulncheck
+vulncheck:
```

Needs a dependency on GOVULNCHECK
```suggestion
vulncheck: $(GOVULNCHECK)
```